_pages/JezBassinder.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: JezBassinder
+pageUrl: JezBassinder.github.io
+timestamp: TODAYS-DATE-2017-06-04
```

Comment:
int line 4: `timestamp: TODAYS-DATE-2017-06-04 ` i think you have to remove the `TODAYS-DATE-`